backend/functions/lib/adapters/gateways/case.assignment.cosmosdb.repository.ts

```diff
@@ -82,39 +110,68 @@ export class CaseAssignmentCosmosDbRepository implements CaseAssignmentRepositor
   }
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  getAssignment(assignmentId: string): Promise<CaseAttorneyAssignment> {
+  getAssignment(assignmentId: string): Promise<CaseAssignment> {
     throw new Error('Method not implemented.');
   }
 
-  async findAssignmentsByCaseId(caseId: string): Promise<CaseAttorneyAssignment[]> {
-    const query = 'SELECT * FROM c WHERE c.caseId = @caseId AND NOT IS_DEFINED(c.unassignedOn)';
+  async getAssignmentHistory(caseId: string): Promise<CaseAssignmentHistory[]> {
+    const query =
+      'SELECT * FROM c WHERE c.documentType = "ASSIGNMENT_HISTORY" AND c.caseId = @caseId ORDER BY c.occurredAtTimestamp DESC';
     const querySpec = {
       query,
       parameters: [
         {
           name: '@caseId',
           value: caseId,
         },
+        {
+          name: '@documentType',
+          value: 'ASSIGNMENT_HISTORY',
+        },
       ],
     };
     const response = await this.queryData(querySpec);
-    return response;
+    return response as CaseAssignmentHistory[];
   }
 
-  async findAssignmentsByAssigneeName(name: string): Promise<CaseAttorneyAssignment[]> {
+  async findAssignmentsByCaseId(caseId: string): Promise<CaseAssignment[]> {
+    const query =
+      'SELECT * FROM c WHERE c.documentType = "ASSIGNMENT" AND c.caseId = @caseId AND NOT IS_DEFINED(c.unassignedOn)';
+    const querySpec = {
+      query,
+      parameters: [
+        {
+          name: '@caseId',
+          value: caseId,
+        },
+        {
+          name: '@documentType',
+          value: 'ASSIGNMENT',
+        },
+      ],
+    };
+    const response = await this.queryData(querySpec);
+    return response as CaseAssignment[];
+  }
+
+  async findAssignmentsByAssigneeName(name: string): Promise<CaseAssignment[]> {
     const querySpec = {
       query: 'SELECT * FROM c WHERE c.name = @name',
       parameters: [
         {
           name: '@name',
           value: name,
         },
+        {
+          name: '@documentType',
```

Comment:
@documentType does not exist as a token in the query. This is static too so it could just be added to the query string.

Reply:
Done in mob branch.